sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/PhoneNumbersClient.java

```diff
@@ -116,6 +160,23 @@ public PurchasedPhoneNumber getPurchasedPhoneNumber(String phoneNumber) {
         return asyncClient.beginPurchasePhoneNumbers(searchId, context).getSyncPoller();
     }
 
+    /**
+     * Starts the update of capabilities for a purchased phone number.
+     *
+     * This function returns a Long Running Operation poller that allows you to wait indefinitely until the
+     * operation is complete.
+     *
+     *
+     * @param phoneNumber The phone number id in E.164 format. The leading plus can be either + or encoded
+     *                    as %2B.
+     * @return A {@link SyncPoller} object.
```

Comment:
Comment here is inconsistent with above, line 101

Reply:
Thanks, done!